ETL.md

```diff
@@ -0,0 +1,144 @@
+Cloud Connection Status has been changed and it should not return to this value. This status is assigned on creation
+to the Cloud Store
+
+- MISSING_CONFIGURATION means that Kubecost has not detected any method of Cloud Configuration. This value is only
+possible on the first Cloud Store that is created as a wrapper for the open source cloud provider. This status
+is assigned during failures in Configuration Retrieval.
+
+- INCOMPLETE_CONFIGURATION means that Cloud Configuration is missing required values to connect to the cloud provider. This status is assigned during failures in Configuration Retrieval.
+
+- FAILED_CONNECTION: means that all required Cloud Configuration values are filled in, but a connection with the
+Cloud Provider cannot be established. This is indicative of a typo in one of the Cloud Configuration values or an
+issue in how the connection was set up in the Cloud Provider's Console. The assignment of this status varies
+between Providers, but should happen if there if an error is thrown when an interaction with an object from
+from the Cloud Service Provider's SDK occurs.
+
+- MISSING_DATA:  means that the Cloud Integration is properly configured, but the cloud provider is not returning
+billing/cost and usage data. This status is indicative of the billing/cost and usage data export of the Cloud Provider
+being incorrectly set up or the export being set up in the last 48 hours and not having started populating data yet.
+This status is set when a query has been successfully made but the results come back empty. If the cloud provider,
+already has a SUCCESSFUL_CONNECTION status then this status should not be set, because this indicates that the specific query made may have been empty.
+
+- SUCCESSFUL_CONNECTION: means that the Cloud Integration is properly configured and returning data. This status is set on any successful query where data is returned
+
+After starting or restarting, each of the Cloud Processes begins two subprocesses, one which fills in historic data over the coverage of the Daily Asset Store and one which run periodically, on a predefined interval, to collect and process new cost and usage data as it is made available by the Cloud Service Provider. The ETL's status endpoint contains a `cloud` object that provides information about each Cloud Store including the Cloud Connection Status and diagnostic information about each of the Cloud Processes. The diagnostic items on the Cloud Processes are:
+
+- Coverage: The window of time that historical subprocess has covered
+- LastRun: The last time that the process ran, updates each time the periodic subprocess runs
+- NextRun: Next scheduled run of the periodic subprocess
+- Progress: Ratio of Coverage to Total amount of time to be covered
+- RefreshRate: The interval that the periodic subprocess runs
+- Resolution: The size of the assets being retrieved
+- StartTime: When the Cloud Process was started
+
+## EndPoints
```

Comment:
How can you just get general diagnostics without taking an action? That seems valuable to me, but maybe I'm mistaken.

Reply:
Good call, I added the endpoint to the doc